plugins/modules/azure_rm_notificationhub.py

```diff
@@ -0,0 +1,360 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2021 Praveen Ghuge (@praveenghuge), Karl Dasan (@karldas30)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+import time
+from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase
+__metaclass__ = type
+DOCUMENTATION = '''
+---
+module: azure_rm_notificationhub
+version_added: "0.1.2"
+short_description: Manage Notification Hub
+description:
+    - Create, update and delete instance of Notification Hub.
+options:
+    resource_groups:
+        description:
+            - Name of the resource group to which the resource belongs.
+    namespace_name:
+        description:
+            - Name of the namespace in which to create notification hub
+    name:
+        description:
+            - Unique name of the app service plan to create or update.
+        required: True
+    location:
+        description:
+            - Resource location. If not set, location from the resource group will be used as default.
+    sku:
+        description:
+            - The name of the SKU.
+            - Please see (https://azure.microsoft.com/en-in/pricing/details/notification-hubs/)
+        default: free
+        choices:
+            - free
+            - basic
+            - premium
+    state:
+      description:
+          - Assert the state of the Notification Hub.
+          - Use C(present) to create or update an notification hub and C(absent) to delete it.
+      default: present
+      choices:
+          - absent
+          - present
+extends_documentation_fragment:
+- azure.azcollection.azure
+
```

Comment:
Please add log_path and log_mode parameter to this module! Thank you very much!
```suggestion
    log_path:
        description:
            - parent argument.
        type: str
    log_mode:
        description:
            - parent argument.
        type: str
```